lib/config/default-config.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @fileoverview Default configuration
+ * @author Nicholas C. Zakas
+ */
+
+"use strict";
+
+//-----------------------------------------------------------------------------
+// Requirements
+//-----------------------------------------------------------------------------
+
+const Rules = require("../rules");
+
+//-----------------------------------------------------------------------------
+// Helpers
+//-----------------------------------------------------------------------------
+
+/*
+ * Because we try to delay loading rules until absolutely necessary, a proxy
+ * allows us to hook into the lazy-loading aspect of the rules map while still
+ * keeping all of the relevant configuration inside of the config array.
+ */
+const defaultRulesConfig = new Proxy({}, {
```

Comment:
Why we named this "Config"? It's an object with built-in rules, not a configuration?

Reply:
Fair point.